resources/event-bus/values.yaml

```diff
@@ -7,7 +7,7 @@ global:
     path: eu.gcr.io/kyma-project
   event_bus:
     dir: develop/
-    version: c157195c
+    version: a4acd811
```

Comment:
This version does not include the changes done for https://github.com/kyma-project/kyma/issues/3491.